tests/feature/test_steps.py

```diff
@@ -489,19 +495,27 @@ def test_step_trace(pytester):
         from pytest_bdd import given, when, scenario
 
         @given('I have a bar')
-        def _():
+        def i_have_bar():
             return 'bar'
 
         @when('it fails')
-        def _():
+        def when_it_fails():
             raise Exception('when fails')
 
-        @scenario('test.feature', 'When step has failure')
-        def test_when_fails_inline():
+        @pytest.fixture
+        def dependency():
+            raise Exception('dependency fails')
+
+        @when("it's dependency fails")
```

Comment:
Typo

Reply:
This grammatical error exists in the existing test - I have fixed it and the other references, as well as a few other grammatical things I found along the way.